modules/administration/pages/hostname-rename.adoc

```diff
@@ -0,0 +1,21 @@
+[[hostname-rename]]
+= Rename Hostname
+
+With the [command]``spacewalk-hostname-rename`` script it is possible to rename the hostname of the {productname} Server.
+The [command]``spacewalk-hostname-rename`` script is part of the [package]``spacewalk-utils`` package.
+
+The only mandatory parameter for the script is the newly configured IP address of the {productname} Server.
+
+
+
+.Procedure: Renaming Hostname
+. Adjust the hostname in advance.
```

Comment:
There's something really jarring about this. If the procedure is "renaming hostname" I don't expect step one to be "you should already have renamed the hostname".
I'm not sure what to do about it, except for describing this procedure in more detail beforehand.

Reply:
As strange as it is, the script only takes care of renaming things at SUSE Manager/Uyuni Server level, but not on DNS/locall resolution level.

Reply:
Yeah, I just think we need to explain that before we launch into the procedure, to avoid confusion.

Reply:
Maybe the confusion comes from the fact that "renaming hostname" is a somewhat generic term. What the "rename-hostname" script is really doing is entering the information of a changed hostname into the database and internal structures of SUSE Manager. So the hostname rename on system level is something the admin has to do on his own like with any regular system. Our script will ensure that SUSE Manager is aware that the host it runs on has been renamed.

Reply:
ok, thanks!  I hope I got it right.